src/pages/home/report/ReportActionsView.js

```diff
@@ -250,8 +249,7 @@ class ReportActionsView extends React.Component {
      * @returns {Boolean}
      */
     getIsReportFullyVisible() {
```

Comment:
I'm thinking we don't need this function anymore, we can just directly call the util right?

Reply:
@hoangzinh I think I like your latest idea best:

> Or we can rename current method `this.getIsReportFullyVisible` to `this.isReportFullyVisible` in `ReportActionView ` in to prevent code boilerplate many places?

Can you do that & also update `getIsReportFullyVisibleUtil` to just `getIsReportFullyVisible` in this file for consistency?

Reply:
Thanks 🙏